acquia/acquia_cms
# ACMS-1706: Make acquia_cms_minimal profile Drupal 10 compatible.

**Motivation**

Fixes #ACMS-1706

**Proposed changes**


**Alternatives considered**


**Testing steps**

- Run composer install
- install site using ./vendor/bin/acms acms:install and select any starter kit
- check all the search functionality is working fine

**Merge requirements**
